aten/src/ATen/core/functional.h

```diff
@@ -34,6 +43,18 @@ inline std::vector<R> fmap(const T& inputs) {
   return r;
 }
 
+template<class F, class InputIt, class OutputIt>
+inline void fmap_to(InputIt first, InputIt last, OutputIt d_first, const F& fn) {
+  while (first != last) {
+    *d_first++ = *first++;
+  }
+}
+
+template<class F, class From, class To>
+inline void fmap_to(const From& inputs, To& outputs, const F& fn) {
```

Comment:
Please add some comment, it is not obvious that `fmap_to` would append to `outputs` rather than rewriting `outputs`. Or better, consider renaming `fmap_to` to `fmap_append_to`.

Reply:
Added comment